### ISSUE TITLE
Success text after submitting survey has left text alignment instead of center alignment

### ISSUE THREAD

**Issue by carstenhag** [Instabug/Instabug-Android#287]
Hello there, our tester noticed that the success text after completing a survey has a wrong alignment on Android.

<img width="386" alt="Bildschirmfoto 2020-03-30 um 10 01 55" src="https://user-images.githubusercontent.com/3776997/77889115-a09f0800-726d-11ea-8c28-2c577604d02f.png">

### Steps to Reproduce the Problem
1. Wait for Net Promoter Score survey to show up (via logic defined via website or via code call)
2. Complete Survey
3. Text is misaligned.

### Expected Behavior
Text is center-aligned

### Actual Behavior
Text is left-aligned

### Instabug integration code
*Make sure to remove your app token.*

### SDK Version
9.0.5

### Android Version
API 28

### Device Model
Not completely sure, likely a S9.

**Comment by carstenhag** [comment]
Just trying this out again with 9.1.0, doesn't occur anymore.  
It would be helpful if you could include *which* bugs were fixed in the release notes.

**Comment by carstenhag** [comment]
<img width="372" alt="Bildschirmfoto 2020-03-31 um 08 47 49" src="https://user-images.githubusercontent.com/3776997/77995564-52047300-732c-11ea-92e3-12d1a8e645e7.png">
Alright, I could verify that this was fixed. Can be closed by me, but my point about the release notes still stand :))

**Comment by mzelzoghbi** [comment]
Hello @carstenhag, Thanks for reaching out. 

Technically we didn't face this issue on 9.0.5 and we didn't know why this happened with you on `9.0.5` and solved on `9.1.0`, maybe it was a caching issue or conflicting resources, But we are happy that everything is working fine now.

Regarding the release notes, you are totally right. We will make sure to link GitHub issues to the release notes in our future releases.

For now, we will close this issue and please let us know if you need any further help.

Thank You!

**Comment by carstenhag** [comment]
Hello @mzelzoghbi. Just today, on launch day of the app, it happened again :D. No worries though, it's not a big deal.

It looks fine on the NPS, but it looks bad on the extended survey. This is on Instabug 9.1.5.

![Screenshot_20200702-072145](https://user-images.githubusercontent.com/3776997/86319910-da4d2980-bc35-11ea-9bb2-513f7400c114.jpg)

**Comment by mzelzoghbi** [comment]
Hello @carstenhag Thanks for reporting this. 
We were able to reproduce the issue & We created a **SNAPSHOT** that should fix the issue, Can you please try it and tell us if it fixed the issue for you?
``` 
implementation 'com.instabug.library:instabug:9.1.6.4-SNAPSHOT'
```
you will need to add these lines to your app **build.gradle** before your dependencies
```
repositories {
    ....
    maven {
        url "https://oss.sonatype.org/content/repositories/snapshots"
    }
}
```
This **SNAPSHOT** is completely safe to use and has no debug code at all.

Thank You 😃

**Comment by carstenhag** [comment]
Hey, with that version it's fixed. Let's hope this is fixed for sure now :D 👯

**Comment by carstenhag** [comment]
Hello @mzelzoghbi again, when do you think will this be released/closed? Thanks in advance!

**Comment by carstenhag** [comment]
Ping to request an update ;).

**Comment by mzelzoghbi** [comment]
So sorry I missed your last message 🙏  This will be included in our next public release scheduled within 2 weeks.

**Comment by carstenhag** [comment]
Pinging again ;)

**Comment by mzelzoghbi** [comment]
Hey @carstenhag, We still didn't launch our latest release, it is in the final testing stages and I'll let you know once it is released. 
Thank You

**Comment by MahmoudAmin39** [comment]
Hi @carstenhag ,
This has been fixed on our latest release `9.1.7`. Please feel free to reopen this issue if you encountered this again.
Thank You :blush:

**Comment by carstenhag** [comment]
Seems to be solved. Thanks!
